notebooks/stable-fast-3d/stable-fast-3d.ipynb

```diff
@@ -0,0 +1,1053 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.            rgb_cond=None,
</pre><p>default values are not used anyway as you define all inputs in example inputs, could you please remove None?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2388/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Removed</p>